### DISCUSSION TITLE
How to Remove Error Check for Git Hub Copilot solutions Tab

### DISCUSSION THREAD

**Discussion by vukhaihoan** [community/community#7143] [Copilot Conversations]
My Vs code Check Error Git Hub Copilot solutions Tab, How to Disable it, Thanks ^^
![image](https://user-images.githubusercontent.com/58582870/139824922-28d58981-2eda-47cd-b155-f612f3748b0b.png)

**Comment by vukhaihoan** [comment]
Any one get this error ?

**Reply by Oblosys** [reply]
Same here. The errors sometimes also linger in the Problems view even after closing the Copilot window.

**Reply by manokaran68** [reply]
For me it disappears after a couple of minutes!

**Comment by KTibow** [comment]
I don't encounter this. What extensions do you have installed? Have you tried disabling everything but Copilot, and see if you still get Copilot output treated like normal code?

**Reply by Oblosys** [reply]
Good call, but it still happens when only the Copilot extension is enabled, and even in a fresh folder with a single .ts file. TypeScript tries to type check the window with the suggestions.

**Reply by KTibow** [reply]
Does Ctrl+Shift+P > Change Language Mode let you configure a default?

**Reply by Oblosys** [reply]
I can change the language mode to plaintext, which removes the errors (and syntax coloring), but it's only for the current window. A new window always starts in the language mode of the currently edited file. I can change the 'GitHub Copilot' association to plaintext, and it persists, but it has no effect on the language mode.

I also tried a clean VS Code install, and only installed the Copilot extension, and still had the same problem. Maybe you have some kind of setting that fixes it? Did you try the Copilot on a TypeScript file? It seems to me a problem with the VS Code integration. If the solutions screen used comments from the current language for the 'Synthesizing ..' and '=====' bits, or use the same method that code lens extensions use (like the 'Accept solution' buttons) it should work just fine.

**Reply by KTibow** [reply]
> If the solutions screen used comments from the current language

That sounds like the easiest fix that the Copilot team could do. Hopefully they can get around to this.

**Reply by Oblosys** [reply]
Yeah, it's an odd blemish on an otherwise very solid product. Anyway, since I never actually use the suggestions window I remapped the key binding so I don't accidentally open it anymore :-) Thanks for your suggestions!
